Gruntfile.js

```diff
@@ -179,9 +179,20 @@ module.exports = function(grunt) {
       },
       deploytest: {
         stderr: false,
-        cmd: 'curl -X DELETE http://admin:pass@localhost:5984/medic-test' +
-             ' && curl -X DELETE http://admin:pass@localhost:5984/medic-audit-test' +
-             ' && kanso push http://admin:pass@localhost:5984/medic-test'
+        cmd: 'echo \'Deleting old dbs..\'' +
+             ' && curl -X DELETE http://admin:pass@localhost:5984/medic-test' +
+             ' && curl -X DELETE http://admin:pass@localhost:5984/medic-test-audit' +
+             ' && curl -X DELETE http://admin:pass@localhost:5984/medic-test-reporter' +
+             ' && echo \'Pushing webapp...\'' +
+             ' && kanso push http://admin:pass@localhost:5984/medic-test' +
+             ' && echo \'Cloning medic-reporter...\'' +
+             ' && rm -rf medic-reporter' +
+             ' && git clone https://github.com/medic/medic-reporter' +
```

Comment:
Not a massive fan of e2e tests requiring a working internet connection to run. If you really need reporter (and it sounds like you do) it would be nice to check to see if it's already on the machine locally before going to the network. I think it makes sense that it you had it checked out it would be at the same child level (ie you could look for `../medic-reporter`).